locales/en.default.schema.json

```diff
@@ -477,6 +477,14 @@
     },
     "announcement-bar": {
       "name": "Announcement bar",
+      "settings": {
+        "auto_rotate": {
+          "label": "Auto-rotate slides"
+        },
+        "change_slides_speed": {
+          "label": "Change slides every"
+        }
+      },
```

Comment:
Just calling out that I agree with duplicating these particular translations for the announcement bar vs reusing the entries from slideshow. 

These are not particularly verbose and even though slideshow code is being used as a layer of abstraction here, slideshow is a fully independent section on its own and I don't know if there's a strong case to rely on its settings here.